src/main/kotlin/skytils/skytilsmod/features/impl/handlers/SpamHider.kt

```diff
@@ -69,6 +69,43 @@ class SpamHider {
     private var abilityUses = 0
     private var lastAbilityUsed = ""
 
+    private var SEA_CREATURES = setOf<String>(
```

Comment:
This should be a val (means its final/constant), not var and the explicit generic is not required.